### DISCUSSION TITLE
New command ideas here!

### DISCUSSION THREAD

**Discussion by howroyd** [howroyd/simonsays#46] [Ideas]
Please use this mega thread to suggest any new action ideas you may have.

I'll get us rolling with what is currently on my whiteboard as TODOs:

* Tornado - spin and yeet combined
* Twerk - crouch and shuffle back and forwards?
* Global talk (radio)
* KONAMI  - what could this do. Needs to be special or related to it's use
* Freeze - freeze on the spot.  Probably implement by counter strafing WASD
* Yoga - look straight up
* Feet - look straight down
* Till Hammer - diagonal look up:right and down:left whilst teabagging

**Comment by howroyd** [comment]
New headcam with press and hold keybind

**Comment by howroyd** [comment]
Hurricane: Spin + Rekt

**Comment by MorbidMorke** [comment]
Feet: if the user is already moving, feet simply flicks the camera down very briefly and straight back up. I wonder if the 'look down' aspect of 'feet' should be longer/more forced? I'm not sure how it's implemented.

**Reply by howroyd** [reply]
This sounds like a bug and one similar/same as I experienced in testing and though I fixed. 
Thank you!

Ref: https://github.com/howroyd/simonsays/issues/60
